pkg/config/execution_unit.go

```diff
@@ -111,3 +113,10 @@ func (cfg ExecutionUnit) GetExecutionUnitParamsAsKubernetes() KubernetesTypePara
 		return params
 	}
 }
+
+const (
+	Lambda     = "lambda"
+	Kubernetes = "kubernetes"
+	Ecs        = "ecs"
+	AppRunner  = "apprunner"
+)
```

Comment:
these types being in here make our config provider specific. What was our reasoning for moving them in here?